apps/website/screens/components/button/specs/ButtonSpecsPage.tsx

```diff
@@ -250,124 +481,950 @@ const sections = [
+                  <tr>
+                    <td>
+                      <Code>primaryDisabledInfoBackgroundColor</Code>
+                    </td>
+                    <td>Container fill:disabled (info)</td>
+                    <td>
+                      <Code>color-blue-100</Code>
+                    </td>
+                    <td>#e6f4ff</td>
+                  </tr>
+                  <tr>
+                    <td>
+                      <Code>primaryDisabledDefaultFontColor</Code>
+                    </td>
+                    <td>Label:disabled (default)</td>
+                    <td>
+                      <Code>color-purple-300</Code>
+                    </td>
+                    <td>#cbacec</td>
+                  </tr>
+                  <tr>
+                    <td>
+                      <Code>primaryDisabledErrorFontColor</Code>
+                    </td>
+                    <td>Label:disabled (error)</td>
+                    <td>
+                      <Code>color-red-300</Code>
+                    </td>
+                    <td>#fe9aa7</td>
+                  </tr>
+                  <tr>
+                    <td>
+                      <Code>primaryDisabledWarningFontColor</Code>
+                    </td>
+                    <td>Label:disabled (warning)</td>
+                    <td>
+                      <Code>color-orange-300</Code>
+                    </td>
+                    <td>#facf9e</td>
+                  </tr>
+                  <tr>
+                    <td>
+                      <Code>primaryDisabledSuccessFontColor</Code>
+                    </td>
+                    <td>Label:disabled (success)</td>
+                    <td>
+                      <Code>color-green-300</Code>
+                    </td>
+                    <td>#acecbe</td>
+                  </tr>
+                  <tr>
+                    <td>
+                      <Code>primaryDisabledInfoFontColor</Code>
+                    </td>
+                    <td>Label:disabled (info)</td>
+                    <td>
+                      <Code>color-blue-300</Code>
+                    </td>
+                    <td>#99d5ff</td>
+                  </tr>
+                </tbody>
+              </DxcTable>
+            ),
+          },
+          {
+            title: "Border",
+            content: (
+              <DxcTable>
+                <thead>
+                  <tr>
+                    <th>Component token</th>
+                    <th>Element</th>
+                    <th>Core token</th>
+                    <th>Value</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td>
+                      <Code>primaryBorderThickness</Code>
+                    </td>
+                    <td>Container</td>
+                    <td>
+                      <Code>border-width-0</Code>
+                    </td>
+                    <td>0</td>
```

Comment:
0rem / 0px instead of 0 for consistency

Reply:
I put `0rem / 0px` but there are many places where >0< appears. We should change it in the future to have consistency.

Reply:
You are right. I will keep an eye on this.

Reply:
Covered here: #2051